### PR TITLE
small fix navbar padding-right, appWrapper padding-bottom

### PR DESCRIPTION
**Pull request by idenispankov**
small fix navbar padding-right, appWrapper padding-bottom

### CODE REVIEW

File: frontend/src/components/app/styledApp.js [hunk 1]
```diff
@@ -1,10 +1,8 @@
-// styled components could go here or in the main App.js
-// awaiting discussion further here
 import styled from 'styled-components';
 
 export const AppContainer = styled.div`
-  padding: 20px 40px 90px;
-  background: #e5e5e5;
+  padding: 20px 40px 80px;
```
Comment (thinklikeadesigner) [inline frontend/src/components/app/styledApp.js:4]: I can see why you did this. The padding isn't equal on the left versus the right side of the header. But the design has small padding on left, and larger padding on right. maybe we should follow the design?

Comment (thinklikeadesigner) [reply frontend/src/components/app/styledApp.js:4]: @McCambley what do you think?

Comment (idenispankov) [reply frontend/src/components/app/styledApp.js:4]: This AppContainer looks good to me according to design.

File: frontend/src/components/header/styledHeader.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ import { Link } from 'react-router-dom';
 
 export const Wrapper = styled.div`
   width: 100%;
-  padding: 12px 40px 12px 20px;
+  padding: 12px 22px 12px 20px;
```
Comment (idenispankov) [inline frontend/src/components/header/styledHeader.js:6]: This change also according to the design.

**Review by idenispankov — COMMENTED** [review]
Please check and merge

**Review by cjmaret — APPROVED** [review]

**Pull request merged**